hokusai/config.yml

```diff
@@ -1,4 +1,6 @@
 project-name: force
 pre-deploy: yarn publish-assets
 post-deploy: .circleci/deploy_event.sh
-hokusai-required-version: ">=0.5.4"
+hokusai-required-version: ">=0.5.8"
```

Comment:
I thought we were recommending a loose version here, `~=0.5` is what we had most apps move to during hokusai checklist update processes. My understanding was that the orb should control the specific version.

Reply:
Yes but this change is not backward compatible... devs would get errors if running on an older Hokusai version

Reply:
The orb will install the latest 0.5.x version